packages/transition/src/collapse.tsx

```diff
@@ -62,6 +62,12 @@ export interface CollapseOptions {
    * @default "auto"
    */
   endingHeight?: number | string
+
+  /**
+   * The Motion element to render as.
+   * @default motion.div
+   */
+  as?: React.ElementType
```

Comment:
I understand this is a work in progress but I thought to add that this should be a motion "element type" since the collapse animation won't work if it just any react element type.